models/edr/system/state.sql

```diff
@@ -0,0 +1,4 @@
+{{ config(materialized='table') }}
```

Comment:
Not sure if `state` is necessarily the best name for it.
Perhaps `metadata` is better?

Reply:
I think `metadata` is better :)